src/util/fd_util_base.h

```diff
@@ -387,6 +387,25 @@ __extension__ typedef unsigned __int128 uint128;
 #define FD_PROTOTYPES_END
 #endif
 
+/* FD_ALIGN: Default alignment according to platform:
+    - avx512     => 64
+    - avx        => 32
+    - noarch128  => 16
+    - noarch(64) =>  8 */
+
+ #if FD_HAS_AVX512
+ #define FD_ALIGN (64UL)
+ #elif FD_HAS_AVX
+ #define FD_ALIGN (32UL)
+ #elif FD_HAS_INT128
+ #define FD_ALIGN (16UL)
+ #else
+ #define FD_ALIGN (8UL)
+ #endif
+
+ /* FD_ALIGNED: shortcut to compiler aligned attribute with default alignment */
+ #define FD_ALIGNED __attribute__((aligned(FD_ALIGN)))
```

Comment:
Happy to move it to ballet (it was originally here because I thought we could use it also for vm/flamenco).

`ushort x` won't be aligned, `FD_ALIGNED ushort x` will be. Useful, e.g., in tests.

Right now in ballet every module has its own random alignment. It's never clear if the dev thought about it, or just copied over from a random point in time. If we build 1 library, we should have 1 way to handle alignment.

Anyway, moving to ballet.